### PR TITLE
Readme_gifs: Added gifs to demo components and descriptions

### PR DESCRIPTION
**Pull request by jtromo**
Description
---
The readme needs to be updated to describe the repo/demonstrate functionality.

What Was Changed
---
Updated the readme to be more readable and added gifs to demo functionality.

Testing
---
1. Verify that the readme looks correct/has no spelling errors

---

Please Review: @Workiva/mobile

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 0

File: README.md [hunk 1]
```diff
@@ -4,30 +4,60 @@
...
+Component | Demo
+--- | ---
+<b>WRadio</b><br> Web-like radio button that can be tied to a group and automatically select/deselect as expected from a normal radio button. The size and color of the outer and inner circle can be changed as well as the highlight. | <img src="Gifs/WSwitchWRadio.gif">
+<b>WSwitch</b><br> Custom switch control emulating Apple's custom switches. Adds the ability to change sizes, colors, and sliding behaviors. | <img src="Gifs/WSwitchWRadio.gif">
+
+### Information Views
+Component | Demo
+--- | ---
+<b>WBadge</b><br> Custom number badge that provides the ability to change color, spacing, shape, location, and how the number displays/expands. | <img src="Gifs/WBadge.gif">
+<b>WSpinner</b><br> Custom progress spinner allowing for custom colors, the addition of images,  | <img src="Gifs/WSpinner.gif">
```
Comment (jordanross-wf) [inline README.md]: should be `addition of images.` (comma => period)

File: README.md [hunk 2]
```diff
@@ -4,30 +4,60 @@
...
+<b>WUserLogoView</b><br> An image used to represent a user. Can use either a name which is hashed into a color or and image that is cropped to fit the view. | <img src="Gifs/WUserLogoView.gif">
+
+### Modal Components
+Component | Demo
+--- | ---
+<b>WActionSheet</b><br> Custom action sheets providing the ability to include a picker view, automatically scale to content, scroll, include a cancel button, and tap to dismiss. | <img src="Gifs/WActionSheet.gif">
+<b>WBanner</b><br>  Static banners that display information over content from the top or bottom. Can be automatically dismissed or on tap. | <img src="Gifs/WBanner.gif">
+<b>WLoadingModal</b><br> Displays a loading view over content. Can dim the background view and will disappear after a set period of time. | <img src="Gifs/WLoadingModal.gif">
+<b>WToast</b><br> Custom toast that displays from the top/bottom or sides and can dismiss on top or a timer. The color, text, and transparency can be customized. | <img src="Gifs/WToast.gif">
```
Comment (jordanross-wf) [inline README.md]: ` and can dismiss on top or a timer` sounds a bit funny. Not sure the best option to change to. Something along the lines of `can be customized to dismiss on tap or via a timer` maybe?

File: README.md [hunk 3]
```diff
@@ -4,30 +4,60 @@
...
+
+### Navigation
+Component | Demo
+--- | ---
+<b>WPagingSelectorControl</b><br> Custom paging control that allows scrolling headers, automatically spaced headers and button actions to display a view controller. | <img src="Gifs/WPagingSelectorControl.gif">
+<b>WPagingSelectorVC</b><br> View controller leveraging the WPagingSelectorControl. | <img src="Gifs/WPagingSelectorVC.gif">
+<b>WSideMenuVC</b><br> Custom side menu drawer. Displays view controllers as cells that can tapped to swap the main view. | <img src="Gifs/WSideMenuVC.gif">
```
Comment (jordanross-wf) [inline README.md]: minor: `that can be tapped`

File: README.md (hunk 2 above)
Comment (jtromo) [inline README.md]: Yea it was supposed to 'tap', but i like yours better.

**Comment by jordanross-wf** [comment]
+10 everything loads correctly and looks good

- [ ] Appropriate reviews.
- [ ] Acceptance criteria met.
- [ ] Edge cases tested.
- [ ] Rosie has run.



Merging into `master`.

**Comment by jordanross-wf** [comment]
@Workiva/release-management-p

**Pull request merged**